docs/source/routing/self-hosted/containerization/azure.mdx

```diff
@@ -0,0 +1,163 @@
+
+    ```text showLineNumbers=false
+    # Use the official Apollo Router Core image as the base.
+    # Set the image tag to the desired router version (e.g. v1.61.0)
+    FROM ghcr.io/apollographql/router:v1.61.0
+
+    # Replace the default router config with the local, customized router.yaml
+    COPY router.yaml /dist/config/router.yaml
+    ```
+
+    The Dockerfile sources the base router image from the GitHub container registry, using the version of router you specify. It then copies your customized `router.yaml` configuration file to overwrite the default router configuration file.
+
+1. From the same local directory, use `docker buildx build` to build a new router image for a specific platform. Choose a name and tag for the image, for example `router:v1.61.0`.
+    
+    ```bash showLineNumbers=false
+    docker buildx build --platform linux/amd64  -t router:v1.61.0 --load .
+    ```
+
+    - The `--load` option loads the built image to `docker images`.
+
+1. Run `docker images` and validate that your router image is in the returned list of images.
+
+## Push router image to container registry
+
+Create an Azure Container Registry as needed, then push your router image to it.
+
+### Create new container registry
+
+If you don't have an existing container registry, create a new one:
+
+1. Log in to the Azure Portal and go to [Container registries](https://portal.azure.com/?quickstart=true#browse/Microsoft.ContainerRegistry%2Fregistries).
+1. Click **Create container registry**.
+1. Fill in the details:
+    - **Subscription**: Select your subscription
+    - **Resource group**: Select existing group or create new
+    - **Registry name**: Enter a unique name for your registry (for example, `myapolloregistry`)
+    - **Location**: Select an appropriate region
+    - **Pricing plan**: Select an appropriate plan
+
+1. Click **Review + create**, then click **Create**. 
+1. Your registry should now be created. Click on your registry to go to its portal page. 
+
+### Log in to container registry
+
+1. In a terminal, sign in to Azure CLI:
+
+    ```bash showLineNumbers=false
+    az login
+    ```
+
+1. Log in and authenticate to your registry (for example, `myapolloregistry`):
+
+    ```bash showLineNumbers=false
+    az acr login --name myapolloregistry
+    ```
+
+### Tag and push image to registry
+
+1. Use `docker tag` to create an alias of the image with the fully qualified path to your registry (for example, `myapolloregistry.azurecr.io`):
+
+    ```bash showLineNumbers=false
+    docker tag router:v1.61.0 myapolloregistry.azurecr.io/router:v1.61.0
+    ```
+1. Push the image to the registry:
+
+    ```bash showLineNumbers=false
+    docker push myapolloregistry.azurecr.io/router:v1.61.0
+    ```
+
+1. Use `az acr repository list` to verify your image is now in the registry:
+
+    ```bash showLineNumbers=false
+    az acr repository list --name myapolloregistry
+    ```
+
+<ExpansionPanel title="Expected output">
```

Comment:
Upon opening this, I was expecting a much longer file. If it's only a single line, might we just show it inline instead of collapsing it?